pinot-core/src/main/java/com/linkedin/pinot/core/data/partition/ByteArrayPartitionFunction.java

```diff
@@ -16,40 +16,39 @@
 package com.linkedin.pinot.core.data.partition;
 
 import com.google.common.base.Preconditions;
-import org.apache.kafka.common.utils.Utils;
+import kafka.producer.ByteArrayPartitioner;
 
 
 /**
  * Implementation of {@link ModuloPartitionFunction} that mimics Kafka's
```

Comment:
Fix the typo in previous checkin, it is implementation of ParititionFunction, not ModuloPartitionFunction